src/components/portrait/portrait.pw.tsx

```diff
@@ -533,26 +534,31 @@ test.describe("Accessibility tests for Portrait component", () => {
     }) => {
       await mount(<PortraitComponent tooltipId={tooltipId} />);
 
-      // color-contrast ignored until we can investigate and fix FE-6245
-      await checkAccessibility(page, undefined, "color-contrast");
+      await waitForAnimationEnd(getDataElementByValue(page, "tooltip"));
+      await checkAccessibility(page);
     });
   });
 
-  [
-    [true, "with"],
-    [false, "without"],
-  ].forEach(([boolVal, renderState]) => {
-    test(`should pass accessibility checks ${renderState} a tooltip, when visibility prop is ${boolVal}`, async ({
-      mount,
-      page,
-    }) => {
-      await mount(
-        <PortraitComponent tooltipMessage="foo" tooltipIsVisible={boolVal} />
-      );
+  test("should pass accessibility checks without a tooltip, when visibility prop is false", async ({
```

Comment:
**praise**: nice one splitting this parameterised test in two to make it easier to read 👍🏼